src/styles/isomer-cms/pages/Editor.module.scss

```diff
@@ -12,6 +12,9 @@
 }
 
 .homepageEditorMain {
+  * { 
+    font-family: "Lato";
+  }
```

Comment:
I think we might want to keep the other previously specified fonts as well, in case the user doesn't have Lato?